database/seed.js

```diff
@@ -0,0 +1,49 @@
+const mongoose = require('mongoose');
+mongoose.connect('mongodb://localhost/shop');
+
+var db = mongoose.connection;
+const faker = require('faker');
+
+db.on('error', console.error.bind(console, 'connection error:'));
+db.once('open', function() {
+  console.log('Connected!');
+});
+
+var itemSchema = mongoose.Schema({
+  productId: {
+    type: Number,
+    unique: true },
+  shop: String,
+  location: String,
+  description: String,
+  image: String,
+  price: Number
+});
+
+var itemList = mongoose.model('itemList', itemSchema);
+
+var createSummary = function() {
+  for (var itemId = 1; itemId <= 100; itemId++) {
+    var newItem = new itemList({
+      productId: itemId, 
+      shop: 'My Shop',
+      location: faker.address.city() + ', ' + faker.address.zipCode(),
+      description: faker.lorem.sentence(), 
+      price: faker.commerce.price(),
+      image: faker.image.fashion()
+    });
+    newItem.save((err, itemId) => {
```

Comment:
this will work just fine, but think about how you might incorporate `promises` and `promises.all()` here for code that was used in a product (this is just a script so it's OK, but if you are running a lot of `asynchronous` functions simultaneously it's a good habit to get into using promises.

Let me know if you'd like any info about this!

Reply:
Thanks Jon, will definitely do some research on incorporate promises here.